linker/Mono.Linker.Steps/BlacklistStep.cs

```diff
@@ -96,17 +96,17 @@ bool IsReferenced (string name)
 			return false;
 		}
 
-		static ResolveFromXmlStep GetExternalResolveStep (EmbeddedResource resource, AssemblyDefinition assembly)
+		protected virtual IStep GetExternalResolveStep (EmbeddedResource resource, AssemblyDefinition assembly)
```

Comment:
I had to expose this due to some tech debt we have left over.  The UnityLinker has it's own "UnityResolveFromXmlStep " which is largely a copy of ResolveFromXmlStep with some changes I have not had a chance to unify.  Anyways, I had to introduce a "UnityBlacklistStep" so that we can use our "UnityResolveFromXmlStep"